```
15:18:35 ERROR: C:/tmp/Bazel/3SovMC5L/external/protobuf/BUILD:579:1: Linking of rule '@protobuf//:internal/_api_implementation.so' failed: msvc_link.bat failed: error executing command 
15:18:35   cd C:/tmp/Bazel/3SovMC5L/execroot/tensorflow-pull-requests-windows-bazel
15:18:35   SET PATH=external/local_config_cc/wrapper/bin
15:18:35   external/local_config_cc/wrapper/bin/msvc_link.bat /DLL /WHOLEARCHIVE -m64 -Xcompilation-mode=opt -Wl,@bazel-out/vc_14_0_x64-py3-opt/bin/external/protobuf/internal/_api_implementation.so-2.params: com.google.devtools.build.lib.shell.BadExitStatusException: Process exited with status 1120
15:18:35 LINK : warning LNK4001: no object files specified; libraries used
15:18:35 LINK : error LNK2001: unresolved external symbol _DllMainCRTStartup
15:18:35 bazel-out/vc_14_0_x64-py3-opt/bin/external/protobuf/internal/_api_implementation.so : fatal error LNK1120: 1 unresolved externals
15:18:35 ERROR: C:/tmp/Bazel/3SovMC5L/external/zlib_archive/BUILD:5:1: output 'external/zlib_archive/libzlib.a' was not created
15:18:35 ERROR: C:/tmp/Bazel/3SovMC5L/external/grpc/BUILD:69:1: output 'external/grpc/libgpr.a' was not created
15:18:35 ERROR: C:/tf_jenkins/home/workspace/tensorflow-pull-requests-windows-bazel/tensorflow/core/BUILD:1112:1: declared output 'tensorflow/core/util/version_info.cc' was not created by genrule. This is probably because the genrule actually didn't create this output, or because the output was a directory and the genrule was run remotely (note that only the contents of declared file outputs are copied from genrules run remotely)
```
I saw the new error message on ci, I can reproduce the same error when using "tensorflow-pull-requests-windows-bazel" as the workspace folder name.
Although the error message is not obvious, the underlying problem is still caused by long path. Is it ok to rename the job name to something like `tf-pr-win-bazel`?
So sorry for the trouble, I will fix this soon. (actually, we can only ease this by not using `\\` to save some characters)

--- a/tensorflow/tools/ci_build/windows/cpu/pip/build_tf_windows.sh
+++ b/tensorflow/tools/ci_build/windows/cpu/pip/build_tf_windows.sh
@@ -11,6 +11,10 @@
 #   - Anaconda3
 # * Bazel windows executable copied as "bazel.exe" and included in PATH.
 
+# All commands shall pass, and all should be visible.
+set -x
+set -e
+
 # This script is under <repo_root>/tensorflow/tools/ci_build/windows/cpu/pip/
 # Change into repository root.
 script_dir=$(dirname $0)

